### DISCUSSION TITLE
Is it preferred to use `connect` or can I set the fk directly in an update/create?

### DISCUSSION THREAD

**Discussion by riha** [prisma/prisma#13404] [Q&A] [answered]
I have an update that includes a foreign key. Should I just update the foreign key in the table or should i use the `connect` syntax? 

What's the difference? Are both optionals available at all times?

**Comment by janpio** [comment, marked as answer]
Connecting gives you more guarantees and checks. The other method we call "unchecked" for exactly that reason. So if you do not have a good reason not to use connecting, do that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
